python/sparkdl/image/imageIO.py

```diff
@@ -181,10 +199,12 @@ def filesToDF(sc, path, numPartitions=None):
     numPartitions = numPartitions or sc.defaultParallelism
     schema = StructType([StructField("filePath", StringType(), False),
                          StructField("fileData", BinaryType(), False)])
-    rdd = sc.binaryFiles(path, minPartitions=numPartitions).repartition(numPartitions)
+    rdd = sc.binaryFiles(
```

Comment:
lol i think autopep8's spacing for function calls doesn't match pep8... but oh well, good enough for me.